debian/rules

```diff
@@ -8,8 +8,10 @@ include /usr/share/dpkg/pkg-info.mk
 
 PYTHON2_VERSIONS = $(shell pyversions -r)
 
+CVERSION := $(shell dpkg-parsechangelog | grep '^Version:' | cut -d' ' -f2 | cut -d- -f1)
```

Comment:
I think you want to use DEB_UPSTREAM_VERSION here (from pkg-info.mk). No need to define this new variable.